webapp/cached.py

```diff
@@ -9,7 +9,9 @@ def __init__(self, *args, **kwargs):
 
     def __call__(self, func):
         def inner(*args, **kwargs):
-            param = args[1]
+            param = args[0]
```

Comment:
does python not have ternaries 😛 ?

Reply:
Updated- there's a ternary operator using same-line if-else statements. I usually prefer standard if-else, but here I guess it makes sense.